client/model_admin_permission_schema.go

```diff
@@ -30,7 +29,8 @@ type AdminPermissionSchema struct {
 	// What level this permission applies to. Either root, project or the name of the environment it applies to
 	Type string `json:"type"`
 	// Which environment this permission applies to
-	Environment *string `json:"environment,omitempty"`
+	Environment          *string `json:"environment,omitempty"`
+	AdditionalProperties map[string]interface{}
```

Comment:
This is what we get with additional properties and it makes it more resilient :clap:

Reply:
> it makes it more resilient

Now that I think about it... does it? Because we saw all tests green for many versions (except for the major upgrade), and I'm (almost) sure we added a field or two to one of the endpoints we're using from the SDK...

Reply:
I'm actually not even sure to be honest but I think the more important change was turning off the `disallowAdditionalPropertiesIfNotPresent` in the PR before the test PR we did. Prior to that, the API didn't work against latest Unleash. 

I know that when additionalProperties is set to false and the models don't match exactly then you get sadness at runtime. And since the projects API goes through two separate flags that change its runtime output... gosh...